src/pages/Login/LoginForm/PasswordTextField.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { TextField } from '@mui/material'
+import { UseFormRegister, FieldErrors } from 'react-hook-form'
+
+import { LoginInput } from './LoginInput'
+
+
+export const PasswordTextField: React.FC<{ errors: FieldErrors<LoginInput>, register: UseFormRegister<LoginInput> }> = ({ errors, register }) => (
+	<TextField
+		id="password-input"
+		label="Password"
+		variant="outlined"
+		margin="normal"
+		type="password"
+		{...register('password', {
+			required: 'Password is Required',
+			pattern: {
+				value: /^(?=.*[A-Za-z])(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9]).{6,20}$/,
```

Comment:
define this regex as const same as above

Reply:
it's easy enough to do. work on it soon

Reply:
done